### PR TITLE
feature(search): adjust styling it to mock dropdown functionality

### PR DESCRIPTION
**Pull request by simonchen92**
#### Summary

- Work Type: Style
- Fixes: 

#### What I did

- Styled search bar to make it look like a drop down
- Made minor changes to code for the end year to display `Present` if it hasn't ended
-

#### Notes / Research / References

-
-

#### Before/After Preview

- Before:

- After:

#### Checklist

- [x] Branch is in format `TYPE/scope/description`
- [x] PR title is in format `TYPE(scope): description`
- [x] Commits are in semantic format
- [x] Has Summary

### CODE REVIEW

**Comment by simonchen92** [comment]
I haven't really finished it yet. Work in progress to keep track of where I am at

**Comment by simonchen92** [comment]
eliminate border on search results because I think it looks better and added on hover effects

File: src/scss/SeriesSearch.scss [hunk 1]
```diff
@@ -1,7 +1,46 @@
...
+    height: 2rem;
+  }
 }
 
-.series-search h1 {
-  font-size: 1rem;
+.search-results-overlay {
+  position: absolute;
+  left: 25vw;
+  background: tomato;
```
Comment (davidholyko) [inline src/scss/SeriesSearch.scss:17]: Probably dont need background here.

File: src/scss/SeriesSearch.scss [hunk 2]
```diff
@@ -1,7 +1,46 @@
...
+  border-radius: 3px;
+  border: none;
+  width: 50vw;
+}
+
+.search-series-suggestion {
+  display: flex;
```
Comment (davidholyko) [inline src/scss/SeriesSearch.scss:24]: i expected to have an alternating background-color for these suggestions. maybe even ones are orang and odd ones are dark orange

File: src/scss/SeriesSearch.scss [hunk 3]
```diff
@@ -1,7 +1,46 @@
...
+    & p {
+      margin: 0 0 0 10px;
+      padding: 0;
+      text-align: left;
+    }
+  }
+}
+
+.search-series-suggestion:hover {
+  background-color: white;
```
Comment (davidholyko) [inline src/scss/SeriesSearch.scss:38]: probably can attach this to css selector above

```
.search-series-suggestion{
  &:hover {
    ...
  }
}

```

File: src/scss/Main.scss [hunk 4]
```diff
@@ -15,3 +15,7 @@ p {
   color: black;
   margin: 0.4em auto;
 }
+
+.d-flex {
```
Comment (davidholyko) [inline src/scss/Main.scss:19]: we might want to have our own bootstrap.scss file instead and lop classes like these into that file. It should be only used for dev purposes.

File: src/components/SeriesSearch/SeriesSearchForm.js [hunk 5]
```diff
@@ -31,27 +31,34 @@ export const SeriesSearchForm = () => {
...
-                </p>
-                <p>
-                  {seriesStartYear}-{seriesEndYear}
-                </p>
-                <img src={imageURL} width={50} />
+                <img
+                  className="search-series-img"
+                  alt="tv show poster image"
+                  src={imageURL}
+                  width={50}
```
Comment (davidholyko) [inline src/components/SeriesSearch/SeriesSearchForm.js:52]: this width property probably should not be set on this element. maybe add it to the class

File: src/components/Main/TvShowPoster.js [hunk 6]
```diff
@@ -4,7 +4,11 @@ import { Link } from 'react-router-dom';
 export const TVShowPoster = ({ data }) => {
   return (
     <Link to={`chart/${data.titleID}`}>
-      <img className="tv-show-poster-sm" src={data.imageURL}></img>
+      <img
+        alt="tv show poster image"
```
Comment (davidholyko) [inline src/components/Main/TvShowPoster.js:8]:
```
alt=`Poster Image of ${data.title}`
```

File: src/scss/SeriesSearch.scss [hunk 7]
```diff
@@ -1,7 +1,54 @@
...
+  left: 25vw;
+  border-radius: 3px;
+  border: none;
+  width: 50vw;
+}
+
+.search-series-suggestion {
+  display: flex;
+  padding: 5px 10px 10px 10px;
+  &:nth-child(even) {
```
Comment (davidholyko) [inline src/scss/SeriesSearch.scss:25]: 👍  this is good. Deciding a color palette will be hard

**Review by davidholyko — APPROVED** [review]
LGTM

**Pull request merged**